pyproject.toml

```diff
@@ -19,3 +100,165 @@ skip = '.git,*.pdf,*.svg,versioneer.py,*.css,test_*'
 check-hidden = true
 ignore-regex = '^\s*"image/\S+": ".*|\b[Mm]anuel[. ][Hh]oltgrewe\b'
 ignore-words-list = 'testin'
+
+####################################################################
+# Pixi Configuration
+####################################################################
+
+[tool.pixi.project]
+channels = ["conda-forge", "bioconda"]
+platforms = ["linux-64", "osx-arm64", "win-64", "osx-64", "linux-aarch64"]
+
+[tool.pixi.pypi-dependencies]
+snakemake = { path = ".", editable = true }
+
+[tool.pixi.environments]
+dev = { features = [
+  # optional dependencies
+  "messaging",
+  "pep",
+  "reports",
+  # plugins
+  "plugin-interfaces",
+  "storage-plugins",
+  "executor-plugins",
+  # dev 
+  "test",
+  "docs",
+  "style",
+] }
+test = { features = [
+  "test",
+  "plugin-interfaces",
+  "storage-plugins",
+  "executor-plugins",
+] }
+docs = { features = ["docs", "plugin-interfaces"] }
+quality = { features = ["style"] }
+publish = { features = ["publish"] }
+
+#----------------------- Plugins -----------------------------
+
+# Plugin interfaces
+[tool.pixi.feature.plugin-interfaces.pypi-dependencies]
+snakemake-interface-executor-plugins = ">=9.3.2, <10"
+snakemake-interface-storage-plugins = ">=3.3.0, <4"
+
+# Storage plugins
+[tool.pixi.feature.storage-plugins.pypi-dependencies]
+snakemake-storage-plugin-http = "*"
+snakemake-storage-plugin-s3 = "*"
+snakemake-storage-plugin-fs = "*"
+
+# Executor plugins
+[tool.pixi.feature.executor-plugins.pypi-dependencies]
+snakemake-executor-plugin-cluster-generic = "*"
```

Comment:
These I don't get. The interfaces are already regular dependencies of snakemake (see above). The plugins are needed to certain tests. Why not just adding them to the test dependencies?